drivers/FlashIAP.cpp

```diff
@@ -56,12 +57,15 @@ int FlashIAP::init()
 {
     int ret = 0;
     _mutex->lock();
+    mbed_mpu_manager_lock_mem_xn();
+
     if (flash_init(&_flash)) {
         ret = -1;
     }
     uint32_t page_size = get_page_size();
     _page_buf = new uint8_t[page_size];
 
+    mbed_mpu_manager_unlock_mem_xn();
```

Comment:
Will it be good to do execute never unlock, before new operation and delete somewhere down?